docs/en/setup/service-agent/java-agent/README.md

```diff
@@ -159,6 +159,10 @@ property key | Description | Default |
 `plugin.kafka.topic_profilings` | Specify which Kafka topic name for Thread Profiling snapshot to report to. | `skywalking_profilings` |
 `plugin.kafka.topic_management` | Specify which Kafka topic name for the register or heartbeat data of Service Instance to report to. | `skywalking_managements` |
 `plugin.springannotation.classname_match_regex` |  Match spring beans with regular expression for the class name. Multiple expressions could be separated by a comma. This only works when `Spring annotation plugin` has been activated. | `All the spring beans tagged with @Bean,@Service,@Dao, or @Repository.` |
+`plugin.grpclog.service_host` | Specify which grpc server's host for log data to report to. | `127.0.0.1` |
+`plugin.grpclog.service_port` | Specify which grpc server's port for log data to report to. | `11800` |
+`plugin.grpclog.max_message_size` | Specify the maximum size of log data for grpc client to report to. | `10485760` |
+`plugin.grpclog.upstream_timeout` | How long grpc client will timeout in sending data to upstream. Unit is second.|`30` seconds|
```

Comment:
Should be `toolkit.log.grpc.reporter.*`

Reply:
Using `@PluginConfig` to set config, isn't the prefix of key  fixed to plugin plus classname?

Reply:
Take a look at this, https://github.com/apache/skywalking/pull/6194#discussion_r558848862. I mean, that should be changed.

Reply:
I'm so sorry to miss it. Do you means using `Toolkit` as the class name and adding `log_grpc_reporter` as the prefix of the field, so the key willi like plugin.toolkit.log_grpc_reporter*? Or using a long class name? Or something else？

Reply:
`toolkit.log.grpc.reporter` means you have deeper nested static class. Not a long class name. 
The nested inner class will be transformed as the key automatically by the agent core.